test/stacks.test.ts

```diff
@@ -1,7 +1,9 @@
 import { expect as expectCDK, haveResourceLike } from '@aws-cdk/assert';
+import { KubernetesVersion } from '@aws-cdk/aws-eks';
 import * as cdk from '@aws-cdk/core';
 import * as ssp from '../lib';
-import { NestedStackAddOn } from '../lib';
+import { ApplicationTeam, AwsLoadBalancerControllerAddOn, ExternalDnsAddon, GlobalResources, ImportHostedZoneProvider, NestedStackAddOn, NginxAddOn } from '../lib';
```

Comment:
`lib` is already imported in line 4. I think import on line 5 can be removed to reduce the verbosity.